project/Build.scala

```diff
@@ -874,6 +874,11 @@ object Build extends sbt.Build {
         ) ++ (
             if (sems.strictFloats) Seq(Tests.Argument("-tstrict-floats"))
             else Seq()
+        ) ++ (
+            if (sems.moduleInit == CheckedBehavior.Compliant)
+              Seq(Tests.Argument("-tcompliant-moduleinit"))
+            else
+              Seq()
```

Comment:
To be consistent, these settings should also come between the `sems.asInstanceOfs` and `sems.strictFloats`.